src/app/components/transaction/pending-label.tsx

```diff
@@ -4,10 +4,15 @@ import { Box, Flex, Stack, Text, color } from '@stacks/ui';
 
 import { Tooltip } from '@app/components/tooltip';
 
-const pendingWaitingMessage =
+const DEFAULT_PENDING_WAITING_MESSAGE =
```

Comment:
We try to use camelCase for constants which are not exported and shared throughout the app.

Reply:
ah, ok, fixed